src/lmms_engine/models/qwen3_moe/monkey_patch.py

```
@@ -0,0 +1,124 @@
+    if cross_entropy:
+        from transformers.loss.loss_utils import nn
+
+        nn.functional.cross_entropy = liger_cross_entropy
+
+    if fused_linear_cross_entropy:
+        if use_rmpad:
+
+            def wrap_forward(func):
+                @wraps(func)
+                def wrapper(*args, **kwargs):
+                    return func(use_rmpad=use_rmpad, *args, **kwargs)
```

Ruff (0.12.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)